host-os-iso-creator.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) IBM Corp. 2016.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import argparse
+import logging
+import yaml
+
+from lib import iso_spinner
+from lib import log_helper
+
+LOG = logging.getLogger(__name__)
+
+
+def main(args):
+    config_file = args['config_file']
+    config = {}
+    with open(config_file) as stream:
+        config = yaml.safe_load(stream)
+
+    log_helper.LogHelper(logfile=config.get('log_file'),
+                         verbose=True),
+    spinner = iso_spinner.MockPungiSpinner(config)
+    spinner.build()
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--config-file', '-c',
+                        help='Path of the configuration file for create iso',
+                        default='./iso_config.yaml')
```

Comment:
We need to find an easier way to have different arguments in different scripts. I don't have any suggestions right now though, so that's fine.